### PR TITLE
Plugner's patch 04 and reverting patch-02 (re-inserting label in command)

### PR DESCRIPTION
**Pull request by Plugner**
Created 3 new classes, to improve the plugin's developer experience, but without affect how the old plugins works.
I'm trying make the same plugin development of bukkit, to make a better migration from bukkit to PowerNukkit.

Files:
.dockerignore - removed circle.yml from ignore, because does not exists.

Classes:

 cn/nukkit/inventory/InventorySize (get the vanilla's inventories size)
 cn/nukkit/ChatColor (Merge the Bukkit's class to work with Nukkit)
 cn/nukkit/ChatColorCharOutOfBoundsException (Exception if the player try to use a < 2 characters as a Color in ChatColor.getByChar

Sorry for my bad english.

### CODE REVIEW

**Comment by Plugner** [comment]
PS: My changes will not affect the normal plugins.
PS2: ChatColor class merged from Bukkit project.

**Comment by Plugner** [comment]
> IMHO, this is a bad decision. Command can have an aliases and plugin may process command differently depending on which alias is used. "label" provides knowledge of the alias currently in use, and removing it disables the feature described previously.

Reverted.

File: src/main/java/cn/nukkit/inventory/InventorySize.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+public class InventorySize {
+    public static final int CHEST = 3 * 9;
+    public static final int DOUBLECHEST = 6 * 9;
+    public static final int DISPENSER = 9;
+    public static final int DROPPER = 9;
+    public static final int HOPPER = 5;
+    public static final int FURNACE = 3;
+    public static final int BREWING_STAND = 4;
+    public static final int BEACON = 3 * 9;
+}
```
Comment (joserobjr) [inline src/main/java/cn/nukkit/inventory/InventorySize.java:12]: This class is not necessary, these information are already available in the `InventoryType` enum.

Example:
```java
InventoryType.DOUBLE_CHEST.getDefaultSize();
```

File: src/main/java/cn/nukkit/exception/ChatColorCharOutOfBoundsException.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package cn.nukkit.exception;
+
+public class ChatColorCharOutOfBoundsException extends Exception {
+    public ChatColorCharOutOfBoundsException(String message) {
+
+    }
```
Comment (joserobjr) [inline src/main/java/cn/nukkit/exception/ChatColorCharOutOfBoundsException.java:6]: This constructor is doing nothing with the `message` parameter, it should be calling `super(message)`.

File: src/main/java/cn/nukkit/ChatColor.java [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+public enum ChatColor {
+    /**
+     * Represents black
+     */
+    BLACK('0', 0x00),
+    /**
+     * Represents dark blue
+     */
+    DARK_BLUE('1', 0x1),
```
Comment (joserobjr) [inline src/main/java/cn/nukkit/ChatColor.java:17]: Nukkit also have a class which does exactly the same. It is the [`TextFormat`](https://github.com/GameModsBR/PowerNukkit/blob/master/src/main/java/cn/nukkit/utils/TextFormat.java) enum.

**Comment by joserobjr** [comment]
Closing as discussed by Discord, thank you for your intent 👍

**Pull request closed without merging**